app/controllers/groups_controller.rb

```diff
@@ -1,30 +1,29 @@
 class GroupsController < ApplicationController
-    before_action :authenticate_user!
+  before_action :authenticate_user!
 
-    def index
-        @groups = current_user.groups.display_groups
-    end
+  def index
+    @groups = current_user.groups.display_groups
+  end
```

Comment:
- [ ] When creating a new category and I provide a valid URL for the input, it is rendered as a string in the Groups#Index page instead of an actual picture. To solve this, I would advise you to use the `img` tag instead of the `p` tag in the `app/views/groups/index.html.erb` file then provide the `src` attribute of the `img` tag with the icon data. 👍 😎 

### How it is rendered

![image](https://github.com/yegonkimutai/budget-app/assets/77106962/af66c276-69b6-4bf8-a927-f8f0a097bb26)